src/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-standard-components",
-  "version": "14.0.0",
+  "version": "14.0.1",
```

```suggestion
  "version": "14.1.0",
```
adding a new method to MojaloopRequests class is a nonbreaking change, so the feature should be bumped, not a patch

---

Ok